src/EditorFeatures/CSharpTest/Completion/CompletionProviders/TypeImportCompletionProviderTests.cs

```diff
@@ -1798,6 +1798,74 @@ namespace Foo
                     inlineDescription: "Foo");
         }
 
+        [WpfFact]
+        public async Task TestAfterScoped()
```

Comment:
please don't write 'completion provider' tests (note: KeywordCOmpletionProvider is fine).  Please put completion tests in CSharpCompletionCommandHandlerTests instead.

Reply:
Thanks @CyrusNajmabadi. That fixed the test failure.